src/App.tsx

```diff
@@ -1,14 +1,73 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useMemo, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { TodosType } from './types/TodosType';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [query, setQuery] = useState('');
+  const [typeOfLoad, setTypeOfLoad] = useState<TodosType | string>(TodosType.All);
```

Comment:
```suggestion
  const [typeOfLoad, setTypeOfLoad] = useState<TodosType>(TodosType.All);
```

Reply:
why do you need string here?

Reply:
![Screenshot (186)](https://user-images.githubusercontent.com/115353346/236161469-8f337da0-0c73-470f-a2ac-fc5af49b51eb.png)

Reply:
If i remove 'string' there, i'm getting error